spec/ios_bump_version_release_spec.rb

```diff
@@ -11,34 +11,17 @@
       other_action_mock = double()
       allow(Fastlane::Action).to receive(:other_action).and_return(other_action_mock)
       allow(other_action_mock).to receive(:ensure_git_branch).with(branch: default_branch)
-
-      allow(Fastlane::Helper::GitHelper).to receive(:checkout_and_pull).with(default_branch)
-      allow(Fastlane::Helper::GitHelper).to receive(:create_branch).with("release/#{next_version_short}", from: default_branch)
-
-      allow(Fastlane::Helper::Ios::VersionHelper).to receive(:get_version_strings).and_return(versions)
-      allow(Fastlane::Helper::Ios::VersionHelper).to receive(:update_xc_configs).with(next_version, next_version_short, nil)
     end
 
     it 'does the fastlane deliver update' do
```

Comment:
The title for this spec example is not really describing what that unit test really checks anymore. You should update this to use a more appropriate unit test description for this one instead.

Reply:
Nice catch, thanks!

Reply:
Done here: 694440b4b187db338c8797515ee9623967a86a2e